.pipelines/build-pipelines.yml

```diff
@@ -214,7 +214,7 @@ steps:
   inputs:
     command: 'push'
     feedsToUse: 'select'
-    packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
+    packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;$(Build.ArtifactStagingDirectory)/**/*.snupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
```

Comment:
If symbols package end in `.snupkg`, do we have any packages with  `*.symbols.nupkg`? If not, do we still need this extension to be pushed as well?

Reply:
There are no packages generated in *.symbols.nupkg format. `*.symbols.nupkg` is the legacy format for symbol packages and is not supported by Nuget.org. Only .snupkg format is supported. 

Actually, packages with the extension `*.symbols.nupkg` format are excluded from pushing (by using the `!` operator in the pattern). This exclusion happens by default with the `NugetCommand@2` pipeline task ( [Docs](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/nuget-command-v2?view=azure-pipelines) ) and is not being added as part of this change.

Reply:
Good catch, I didnt see the `!` operator. If the exclusion happens by default, do we still need to explicitly mention it here? If not required, can we remove it and let the default exclusion be applied?

I realize its not your addition, I'm trying to simplify how we build for symbols.

Reply:
What I wanted to convey by default exclusion is that the `packagesToPush` input field for `NugetCommand@2` task has a default value of `$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg` which excludes the push action for files of the pattern `*.symbols.nupkg`. 

Apologies for not wording my response clearly.

I do see the line of thought ~ Since, there are no packages of the format `*.symbols.nupkg`, removing  `!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg` shouldn't hurt. I think we can have it explicitly just to err on the side of caution. Please let me know your thoughts

Reply:
Since the doc already have this defaulted -  to not include *.symbols.nupkg, we can safely remove it from here. Or does the task definition change over time?

Reply:
`!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg`, @Shyam, can you check in the artifact stagging directory if any file with this extension is generated, if not we can remove this statement.

Reply:
Thanks for the suggestions everyone! I checked the artifactory and there are no files with *.symbols.nupkg extension. I have removed the pattern.